static/app/views/organizationCreate.tsx

```diff
@@ -24,10 +26,17 @@ function OrganizationCreate() {
           submitLabel={t('Create Organization')}
           apiEndpoint="/organizations/"
           apiMethod="POST"
-          onSubmitSuccess={data => {
+          onSubmitSuccess={(createdOrg: OrganizationSummary) => {
+            let nextUrl = normalizeUrl(
+              `/organizations/${createdOrg.slug}/projects/new/`,
+              {forceCustomerDomain: true}
+            );
```

Comment:
Should we only call normalizeUrl() if the organization has the customer-domains feature?

Reply:
Oops. `forceCustomerDomain` should only be true only whenever the created organization has the customer domain feature.